pkg/util/ovs/ovs.go

```diff
@@ -138,13 +142,23 @@ func New(execer exec.Interface, bridge string, minVersion string) (Interface, er
 	return ovsif, nil
 }
 
-func (ovsif *ovsExec) exec(cmd string, args ...string) (string, error) {
+func (ovsif *ovsExec) execWithStdin(cmd string, stdinArgs []string, args ...string) (string, error) {
 	if cmd == OVS_OFCTL {
 		args = append([]string{"-O", "OpenFlow13"}, args...)
 	}
-	glog.V(4).Infof("Executing: %s %s", cmd, strings.Join(args, " "))
 
-	output, err := ovsif.execer.Command(cmd, args...).CombinedOutput()
+	kcmd := ovsif.execer.Command(cmd, args...)
+	if stdinArgs != nil {
+		stdinString := strings.Join(stdinArgs, "\n")
+		stdin := bytes.NewBufferString(stdinString)
+		kcmd.SetStdin(stdin)
+
+		glog.V(4).Infof("Executing: %s %s |\n%s", cmd, strings.Join(args, " "), stdinString)
```

Comment:
The use of `|` there is weird: it makes it look like you're piping the output of ovs-ofctl to the string.
Actually, just changing `|` to `<<` should make it clear what's going on, shell-syntax-wise, I think?